source/gamefunctions_debug.bmx

```diff
@@ -1336,7 +1336,7 @@ Type TDebugScreen
 	Method ResetMode_Misc()
 		FastForward_Continuous_Active = False
 		FastForward_Active = False
-		FastForwardSpeed = 500
+		FastForwardSpeed = GameRules.devConfig.GetInt(New TLowerString.Create("DEV_AI_GAME_SPEED"), 500)
```

Comment:
Da kannst Du gerne den default auf 1000 setzen - bzw in der DEV auf 500 .... so zwecks "einheitlich"